.travis.yml

```diff
@@ -1,40 +0,0 @@
-dist: bionic
-
-addons:
-  chrome: stable
-
-language: ruby
-
-matrix:
-  include:
-    - rvm: 2.7.0
-      env: "RAILS_VERSION=6.0.2.2"
-    - rvm: 2.6.5
-      env: "RAILS_VERSION=6.0.2.2"
-    - rvm: 2.6.5
-      env: "RAILS_VERSION=5.2.4.2"
-    - rvm: 2.5.7
-      env: "RAILS_VERSION=5.2.4.2 BLACKLIGHT_API_TEST=true ENGINE_CART_RAILS_OPTIONS=\"--api --skip-git --skip-bundle --skip-listen --skip-spring --skip-yarn --skip-keeps --skip-action-cable --skip-coffee --skip-test\""
-    - rvm: jruby-9.2.8.0
```

Comment:
I don't want to advocate for retaining it, but just checking that testing on jruby is being dropped deliberately. It might be good to call that out in the PR description.

Reply:
The JRuby build has been broken... for a long time.